draft-ietf-secevent-http-push.xml

```diff
@@ -172,8 +172,8 @@
             </t>
             <t>
                 SET Transmitters MAY transmit SETs issued by another entity. The SET
-                Recipient may accept or reject (i.e., return an <spanx style="verb">access_denied</spanx>
-                error response) at its own discretion.
+                Recipient may accept or reject (i.e., return an error response such as
+                <spanx style="verb">access_denied</spanx>) at its own discretion.
```

Comment:
Add "a SET" before "at its own discretion".  Currently the sentence doesn't say what it is that would be accepted or rejected.

Reply:
done